tests/upload_test.go

```diff
@@ -68,10 +81,39 @@ var _ = Describe("[rfe_id:138][crit:high][vendor:cnv-qe@redhat.com][level:compon
 		Expect(deleted).To(BeTrue())
 	})
 
-	table.DescribeTable("should", func(validToken bool, expectedStatus int) {
+	table.DescribeTable("should", func(validToken bool, expectedStatus int, isBlockVolume bool) {
+
+		var pod *v1.Pod
+		var err error
+
+		if isBlockVolume {
```

Comment:
So instead of making the DescribeTable a complicated message based on several booleans, it would make more sense to simply make a separate
```go
var _ = Describe("Upload block volume", func() {
})
```
Where in the before each you create the block PVCs. 

DescribeTables are meant for the case where you have a few parameters to a function, and the expected output changes based on those parameters. Its easy to pass in the different parameters and expected output. But if the test changes completely with certain parameters then it is much better to write a separate test for those parameters.